src/components/Footer.js

```diff
@@ -1,7 +1,19 @@
 import "./Footer.css";
 import { NavLink, Switch, Route } from "react-router-dom";
 
-export default function Footer() {
+export default function Footer({
+  onEnterClick,
+  isRoomSelected,
+  onSubmitClick,
+}) {
+  function handleEnterClick() {
+    onEnterClick();
+  }
+
+  function handleSubmitClick() {
+    onSubmitClick();
+  }
```

Comment:
I don't think you need an inBetween function here since you are not trying to pass an event between components.

I'll suggest some changes down the line:

Reply:
> Hey Patrik,
> 
> I really like what I saw :) Keep up the good work

Thanks a lot for the detailed review!:)